frontend/src/features/collection/state/collectionState.ts

```diff
@@ -1,14 +1,15 @@
 import { RootState } from '../../../store/store';
 import { createReducer } from '@reduxjs/toolkit'
-import { datasetAliasFromWorkflowInfo, WorkflowInfo } from '../../../qrimatic/workflow';
+import { datasetAliasFromVersionInfo, VersionInfo } from '../../../qri/versionInfo';
 
 export const WORKFLOW_STARTED = 'WORKFLOW_STARTED'
 export const WORKFLOW_COMPLETED = 'WORKFLOW_COMPLETED'
 
-export const selectCollection = (state: RootState): WorkflowInfo[] => {
-  const { collection, running, ids } = state.collection
+export const selectCollection = (state: RootState): VersionInfo[] => {
+  const { collection, running } = state.collection
+  const ids = Object.keys(collection)
```

Comment:
no need to declare this constant. Save an allocation by changing line 22 from `ids.forEach` -> `Object.keys(collection).forEach`